docs/kubernetes/monitoring.md

```diff
@@ -65,6 +66,16 @@ Once you have opened the UI, you can explore node stats (CPU, Memory, etc...) un
 
 ![Image of Kuberentes dashboard](../images/k8s-monitoring-dashboard.png)
 
+## Monitoring extension
+
+A quick way to scaffold out cloud-native and open source monitoring components is to use the [acs-engine monitoring extension](https://github.com/Azure/acs-engine/tree/master/extensions/prometheus-grafana-k8s). For details on how to use the monitoring extension, please refer to the extension documentation. By embedding the extension in your apimodel, the extension will do much of the work to create a monitoring solution in your cluster, which includes the following...
+
+- [cAdvisor](https://github.com/google/cadvisor) daemon set to publish container metrics
+- [Prometheus](https://prometheus.io/) for metrics collection and storage
+- [Grafana](https://grafana.com/) for dashboard and visualizations
+
+The extension wires up these components together. Post-deplyoment of the Kubernetes cluster, you just have to retrieve Grafana username password secret and target your browser to the Grafana endpoint. There is already a pre-loaded Kubernetes cluster monitoring dashboard, so out-of-the-box you will have meaningful monitoring points with the extensibility that Prometheus and Grafana offer you.
```

Comment:
`you just have to retrieve Grafana username password secret and target your browser` are there some missing commas here between username, password and secret?

Reply:
I'll reword this so it is more clear. Thanks!